streetcrm/static/js/inline_ajax.js

```diff
@@ -430,10 +435,21 @@ function createPopup(model_id) {
 */
 function closePopup() {
   $("#participantModal").css("display", "none");
+  $("#participantModal").unbind("click");
+  $("#participantModal .modal-content").unbind("click");
```

Comment:
What do these new lines do?

Reply:
Those lines remove the click handlers to avoid that weird duplication bug in #322. Otherwise there's some weird behavior with closing and reopening

Reply:
Aha, thank you.